bare-metal/container-linux/kubernetes/profiles.tf

```diff
@@ -40,21 +61,21 @@ data "template_file" "container-linux-install-configs" {
 }
 
 // Container Linux Install profile (from matchbox /assets cache)
-// Note: Admin must have downloaded container_linux_version into matchbox assets.
+// Note: Admin must have downloaded os_version into matchbox assets.
 resource "matchbox_profile" "cached-container-linux-install" {
```

Comment:
You can skip making a flatcar-cached-install profile for now. Without the standard Matchbox setup having an analog of https://github.com/coreos/matchbox/blob/master/scripts/get-coreos, I don't think users will prepare those assets correctly and it'll lead to issues being filed. Let's consider it in a followup.